### DISCUSSION TITLE
v1.1.0 Full node Docker image

### DISCUSSION THREAD

**Discussion by ZuluSpl0it** [classic-terra/core#192] [Ideas]
I've been trying to get a full node up and running for several weeks now. Just when I think I got it... I get hit with the dreaded "wrong Block.Header.AppHash". 

Error: error during handshake: error on replay: wrong Block.Header.AppHash.  Expected 3C3840FCFD836743A333ECC339D78B7F89FCCD3DB252C1640801176A59F732D3, got 6817B31A67A674548AF44C32D7F18655EBE6B1803CC2A6AA19065452DDA06A67

I've checked and asked for help on Discord: https://discord.com/channels/978124265482231809/1013464023586582528 and https://discord.com/channels/984128262311645254/984128262768852994. There are plenty of others that have the same issue. Some have been able to resolve it while others (including myself) have not. I do have to thank the people that have offered to help and have given suggestions for fixes. After many hours downloading snapshots, addrbooks, and genesis files. Cloning repos, building Terrad, switching out versions (tags), deleting the data folder, re-extracting the snapshots, trying several versions of go - I finally had to give up. This is the first chain that I haven't been able to get a (bare metal) node up and running. 

To help the community support the ecosystem, I'd like to ask the L1 team to produce a docker image that runs v1.1.0 with necessary dependencies, genesis file, addrbook. The app.toml & config.toml files could also be setup so that the docker container spins up and is ready to start syncing. I understand that a validator would need extra configuration, but just having a full node setup working would make it easier for validators to get going.

Perhaps, a dev could take the existing docker setup for the local node and make the necessary changes to get v1.1.0 running on it. I'm also willing to help under the direction of an L1 team member.

**Comment by nghuyenthevinh2000** [comment]
yes, I am working on it. Please hold a little bit

**Comment by nghuyenthevinh2000** [comment]
currently, it is in this folder: https://github.com/classic-terra/core/tree/main/contrib/terra-operator

I am refactoring our Docker assets. Currently building for testnet. But, I will check for mainnet soon

The document needs update also, I will try to test it for mainnet by the end of this week

Docker images will be hosted on here: https://gallery.ecr.aws/classic-terra

**Comment by tlffibra** [comment]
Goodnight! I have been facing the same sync error for my node (TerraV1)
I have already performed tests on several blockchains, for example: (TerraV2/HUB COSMOS), the nodes synchronize perfectly on Terrav2 and HUB Cosmos, today I configured a node on TerraV1 testnet, and the synchronization was perfect.
I only used genesis.json and addrbook.json , I downloaded snaphot.

**Comment by nghuyenthevinh2000** [comment]
check this one: https://github.com/classic-terra/core/pull/197

guide here: https://github.com/classic-terra/core/blob/mainnet-docker/contrib/terra-operator/mainnet-validator/README.md

I have tested the image, it is running on my local machine for 2 days now.

@tlffibra @ZuluSpl0it

**Comment by tlffibra** [comment]
Good afternoon!
nghuyenthevinh2000, for testing purposes, if you want to access my machine and check the error below.

INF Replay last block using real app module=consensus
Error: error during handshake: error on replay: wrong Block.Header.AppHash. Expected 3DE16 2C50515DF4447BF43631DF63864EB41CE643430A8E74F97050837BBD71B, got FC1EEB237CB35B76CA61E15388 04A7FE8616826D1C416512BADA7E97FCE9BAF1

If you want to send me a DM, to pass the IP AND login of the machine, my https://twitter.com/JooBoldrini5
Discord jpboldrini#0243

**Reply by nghuyenthevinh2000** [reply]
I have sent dm to your twitter

**Reply by nghuyenthevinh2000** [reply]
@tlffibra can you confirm here that the situation has been resolved
